src/networks.json

```diff
@@ -273,5 +273,25 @@
     "rpc": ["https://rpc.rupx.io"],
     "ws": ["wss://ws.rupx.io"],
     "explorer": "http://scan.rupx.io"  
+  },
+  "1666600000": {
+    "key": "1666600000",
+    "name": "Harmony Mainnet",
+    "shortName": "HarmonyMainnet",
+    "chainId": 1666600000,
+    "network": "mainnet",
+    "rpc": ["https://api.s0.t.hmny.io"],
```

Comment:
Can you confirm than this is an archive node?

Reply:
sure. it's an archive rpc interface